src/spas/libs/member-management/domain/src/lib/user.mock.service.ts

```diff
@@ -0,0 +1,17 @@
+import { Injectable } from '@angular/core';
+import { User } from './models/user-profile';
+import { USERS } from './mocks/mock-users';
+import { Observable, of } from 'rxjs';
+
+@Injectable({
+  providedIn: 'root'
+})
+
+export class UserMockService {
```

Comment:
mock service should be renamed to ```MemberManagementMockService```, also file to member-management.service.mock.ts

Reply:
oh and mock service should implement the ```MemberManagement``` interface